eng/pipelines/templates/stages/archetype-sdk-data.yml

```diff
@@ -3,6 +3,9 @@ parameters:
   AdditionalModules: []
   SDKType: data
   ServiceDirectory: not-specified
+  TargetDocRepoOwner: 'Azure'
```

Comment:
Can't we default these in the release template yml instead of duplicating everywhere?

Reply:
Unfortunately, I want this thing to properly pull arguments all the way in from the `ci.yml`. If I set default parameter in `archtype-sdk-release`, but still have the parameter passed INTO the release template from `stages/archetype-sdk-*`, it'll get defaulted to empty argument. 

Now, there are arguments to be made for saying
> well then just set the parameter when we actually need this behavior

But I feel like the way we have it set up now is the least amount of duplication when we actually start passing in different stuff from the `ci.yml`.